R/vault.R

```diff
@@ -135,6 +135,7 @@ get_secret_file <- function(vault, name) {
 }
 
 get_secret_user_file <- function(vault, name, email) {
+  if (is.null(email)) return(NULL)
```

Comment:
You're right, this is not needed, it was added to propagate NULL through, but isn't necessary.